spectrometer.cabal

```diff
@@ -95,38 +92,36 @@ common deps
     , http-types                   ^>=0.12.3
     , lzma                         ^>=0.0.0.3
     , lzma-conduit                 ^>=1.2.1
-    , megaparsec                   ^>=9.1.0
+    , megaparsec                   ^>=9.2.0
     , modern-uri                   ^>=0.3.4
     , mtl                          ^>=2.2.2
     , network                      ^>=3.1.2.0
-    , optparse-applicative         >=0.15     && <0.17
+    , optparse-applicative         ^>=0.17.0.0
```

Comment:
This update has help text changes, can you run the main help and the help for `analyze`, and post them in the PR?

Reply:
```
➜ cabal run fossa -- --help        
Up to date
fossa-cli - Flexible, performant dependency analysis

Usage: fossa COMMAND [-V|--version]

Available options:
  -h,--help                Show this help text
  -V,--version             show version information and exit

Available commands:
  analyze                  Scan for projects and their dependencies
  test                     Check for issues from FOSSA and exit non-zero when
                           issues are found
  report                   Access various reports from FOSSA and print to
                           stdout. Currently available reports: (attribution)
  container                Run in container-scanning mode
  list-targets             List available analysis-targets in a directory
                           (projects and sub-projects)
  experimental-link-user-defined-dependency-binary
                           Link one or more binary fingerprints as a
                           user-defined dependency
```

```
➜ cabal run fossa -- analyze --help
Up to date
Usage: fossa analyze [--debug] [-e|--endpoint URL] [-p|--project ARG] 
                     [-r|--revision ARG] [--fossa-api-key ARG] [-c|--config ARG]
                     [--with-telemetry-scope ARG] [-o|--output] 
                     [--unpack-archives] [--json] [--include-unused-deps] 
                     [--force-vendored-dependency-scan-method METHOD] 
                     [--force-vendored-dependency-rescans] [-b|--branch ARG] 
                     [-t|--title ARG] [-P|--project-url ARG] 
                     [-j|--jira-project-key ARG] [-L|--link ARG] [-T|--team ARG]
                     [--policy ARG] 
                     [--release-group-name ARG --release-group-release ARG] 
                     [--only-target PATH] [--exclude-target PATH] 
                     [--only-path PATH] [--exclude-path PATH] 
                     [--experimental-enable-vsi] 
                     [--experimental-enable-binary-discovery] 
                     [--experimental-link-project-binary DIR] 
                     [--experimental-analyze-dynamic-deps BINARY] 
                     [--experimental-skip-vsi-graph LOCATOR] 
                     [--experimental-enable-monorepo MODE] [DIR]

  Scan for projects and their dependencies

Available options:
  --debug                  Enable debug logging, and write detailed debug
                           information to `fossa.debug.json`
  -e,--endpoint URL        The FOSSA API server base URL (default:
                           https://app.fossa.com)
  -p,--project ARG         this repository's URL or VCS endpoint (default: VCS
                           remote 'origin')
  -r,--revision ARG        this repository's current revision hash (default: VCS
                           hash HEAD)
  --fossa-api-key ARG      the FOSSA API server authentication key (default:
                           FOSSA_API_KEY from env)
  -c,--config ARG          Path to configuration file including filename
                           (default: .fossa.yml)
  --with-telemetry-scope ARG
                           Scope of telemetry to use, the options are 'full' or
                           'off'. (default: 'full')
  -o,--output              Output results to stdout instead of uploading to
                           fossa
  --unpack-archives        Recursively unpack and analyze discovered archives
  --json                   Output project metadata as json to the console.
                           Useful for communicating with the FOSSA API
  --include-unused-deps    Include all deps found, instead of filtering
                           non-production deps. Ignored by VSI.
  --debug-no-discovery-exclusion
                           Ignore filters during discovery phase. This is for
                           debugging only and may be removed without warning.
  --force-vendored-dependency-scan-method METHOD
                           Force the vendored dependency scan method. The
                           options are 'CLILicenseScan' or 'ArchiveUpload'.
                           'CLILicenseScan' is usually the default unless your
                           organization has overridden this.
  --force-vendored-dependency-rescans
                           Force vendored dependencies to be rescanned even if
                           the revision has been previously analyzed by FOSSA.
                           This currently only works for CLI-side license scans.
  -b,--branch ARG          this repository's current branch (default: current
                           VCS branch)
  -t,--title ARG           the title of the FOSSA project. (default: the project
                           name)
  -P,--project-url ARG     this repository's home page
  -j,--jira-project-key ARG
                           this repository's JIRA project key
  -L,--link ARG            a link to attach to the current build
  -T,--team ARG            this repository's team inside your organization
  --policy ARG             the policy to assign to this project in FOSSA
  --release-group-name ARG the name of the release group to add this project to
  --release-group-release ARG
                           the release of the release group to add this project
                           to
  --only-target PATH       Only scan these targets. See targets.only in the
                           fossa.yml spec.
  --exclude-target PATH    Exclude these targets from scanning. See
                           targets.exclude in the fossa.yml spec.
  --only-path PATH         Only scan these paths. See paths.only in the
                           fossa.yml spec.
  --exclude-path PATH      Exclude these paths from scanning. See paths.exclude
                           in the fossa.yml spec.
  --experimental-enable-vsi
                           Analyzes project files on disk to detect vendored
                           open source libraries (used for C/C++ support)
  --experimental-enable-binary-discovery
                           Reports binary files as unlicensed dependencies
  --experimental-link-project-binary DIR
                           Links output binary files to this project in FOSSA
  --experimental-analyze-dynamic-deps BINARY
                           Analyzes dynamically linked libraries in the target
                           binary and reports them as dependencies
  --experimental-skip-vsi-graph LOCATOR
                           Skip resolving the dependencies of the given project
                           in FOSSA
  --experimental-enable-monorepo MODE
                           scan the project in the experimental monorepo mode.
                           Supported modes: aosp
  DIR                      Set the base directory for scanning (default: current
                           directory)
  -h,--help                Show this help text

Global options:
  -V,--version             show version information and exit
```